Classes/errortype.py

```diff
@@ -0,0 +1,63 @@
+class ErrorType:
+
+    def __init__(self, name, description):
+        self.name = name
+        self.description = description
+
+
+errors = set([ErrorType('AssertionError', 'Raised when the assert statement fails.'),
```

Comment:
This is messy at the moment. Next step will be to create a json or csv file with all the necessary data.